api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -400,10 +442,13 @@ private boolean userHasEverLoggedIn(User googleUser, DbUser user) {
   private ResponseEntity<Void> resetPasswordAndSendWelcomeEmail(String username, DbUser user) {
     User googleUser = directoryService.resetUserPassword(username);
     try {
-      mailServiceProvider
-          .get()
-          .sendWelcomeEmail(user.getContactEmail(), googleUser.getPassword(), user.getUsername());
-    } catch (MessagingException e) {
+      institutionService
+          .getByUser(user)
+          .ifPresent(
```

Comment:
warning here too

Reply:
Should it be a warning or an error. Since this is during the creating account, user Should always have an institution